docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -73,39 +97,194 @@ const sessionSigsFunction = async() => {
       },
     ],
     authNeededCallback,
-    });
+  });
 
   return sessionSigs;
 };
 ```
 
 **Note:** The nonce should be the latest Ethereum blockhash returned by the nodes during the handshake.
 
-:::note
- If running the SDK in a Server environment, session signatures may *not* be cached unless you provide an instance of `Storage` to the runtime.
- [Here](https://www.npmjs.com/package/node-localstorage) is an implementation of `LocalStorage` which creates local files to persist storage data.
- If storage is not available, session keys *MUST* be persisted in an external data store. 
- ```javascript
- const LocalStorage = require('node-localstorage').LocalStorage;
- const litNodeClient = new LitNodeClient({
-  litNetwork: "datil-dev",
-  debug: true,
-  storageProvider: {
-    provider: new LocalStorage('./storage.test.db'),
-  }
- });
- ```
-:::
+## getPkpSessionSigs()
+This is an implementation of `getPkpSessionSigs()`. Noticably different from `getSessionSigs()`, it lacks the `authNeededCallback` parameter, instead creating the callback and signing the session key under the hood with the `signSessionKey()` function. [API Docs](https://v6-api-doc-lit-js-sdk.vercel.app/classes/lit_node_client_src.LitNodeClientNodeJs.html#getLitActionSessionSigs)
```

Comment:
can we specify why someone would want to use one of these functions over the other? Like hint at recommended usage ('we recommend using this function if you are...') perhaps in the overview at the beginning of this section?

Reply:
@a1ttech Changed this, would appreciate confirmation on the accuracy of my comments 😅:

We recommend using `getSessionSigs()` to specify the abilities of your current session on the Lit Network, using only an `ethers.Wallet` and the `LitNodeClient`. This method is arguably the simplest, enabling your `ethers.Wallet` to perform the specified abilities on the Lit Network. `getPkpSessionSigs()` can be used if your implementation of the Lit SDK involves a PKP and you wish to designate resources to it. Similarly, `getLitActionSessionSigs()` can designate specific abilities and resources to Lit Actions, potentially allowing them to execute code on the Lit Network.